zerver/openapi/zulip.yaml

```diff
@@ -1798,6 +1798,83 @@ paths:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - push_account
+                                op:
+                                  type: string
+                                  enum:
+                                    - update
+                                push_account_id:
+                                  type: string
+                                  description: |
+                                    The push account ID whose registration status changed.
+                                status:
+                                  type: string
+                                  description: |
+                                    The updated registration status.
+                              example:
+                                {
+                                  "id": 1,
+                                  "type": "push_account",
+                                  "op": "update",
+                                  "push_account_id": "1234",
+                                  "status": "active",
+                                }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent to a user's clients when registration to bouncer fails.
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - push_account
+                                op:
+                                  type: string
+                                  enum:
+                                    - remove
+                                push_account_id:
+                                  type: string
+                                  description: |
+                                    The push account ID.
+                                reason:
+                                  type: string
+                                  nullable: true
+                                  description: |
+                                    If the registration fails due to invalid `bouncer_public_key`
+                                    provided in [`POST /mobile_push/register`](/api/register-push-account),
+                                    endpoint, this field is set to `"Invalid bouncer_public_key"`,
+                                    otherwise `null`.
```

Comment:
We should fix this in the code too, but this should not be a human-readable potentially translated, it should be the error `code` value. Maybe `error_code` would be a better name for this field than `reason`.

Reply:
Done.